app/assets/javascripts/src/Metamaps.Header.js

```diff
@@ -0,0 +1,22 @@
+var Metamaps = Metamaps || {}
+
+Metamaps.Header = {
+  init: function () {
+
+  },
+  fetchUserThenChangeSection: function (signedIn, mapperId) {
+    $.ajax({
+       url: '/users/' + mapperId + '.json',
+       success: function (response) {
+         Metamaps.Header.changeSection(signedIn, 'mapper', response.image, response.name)
+       },
+       error: function () {}
+    }); 
+  },
+  changeSection: function (signedIn, section, userAvatar, userName) {
+    ReactDOM.render(
```

Comment:
@devvmh this is where I wrap up the React rendering in a function within the relevant module